src/java.base/share/classes/java/util/WeakHashMap.java

```diff
@@ -428,7 +434,7 @@ public boolean containsKey(Object key) {
         Entry<K,V>[] tab = getTable();
         int index = indexFor(h, tab.length);
         Entry<K,V> e = tab[index];
-        while (e != null && !(e.hash == h && eq(k, e.get())))
+        while (e != null && !(e.hash == h && matchesKey(e, k))
```

Comment:
This doesn't compile, which is why the checks from the GitHub actions build failed.

Reply:
I caught that and it's in my local repo that I haven't pushed the commit.